environment/state.go

```diff
@@ -9,7 +9,26 @@ import (
 )
 
 type State struct {
-	Container string `json:"container"`
+	Container string    `json:"container,omitempty"`
+	Title     string    `json:"title,omitempty"`
```

Comment:
this is the right place for this (goes into notes) but it's kinda weird that it comes from environment.Update (where all the other args go into EnvironmentConfig->git)

Reply:
Agreed. I don't have a better solution in mind right now though, suggestions?